cloudlink-function-mapper-desktop/pom.xml

```diff
@@ -0,0 +1,84 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.gluonhq</groupId>
+    <artifactId>cloudlink-function-mapper-desktop</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
+    </properties>
+    <repositories>
+        <repository>
+            <id>gluon-private-snapshot</id>
+            <url>https://nexus.gluonhq.com/nexus/content/repositories/snapshots/</url>
+        </repository>
+        <repository>
+            <id>Gluon-Public-Snapshot</id>
+            <url>https://nexus.gluonhq.com/nexus/content/repositories/public-snapshots/</url>
+        </repository>
+    </repositories>
+    <dependencies>
+        <dependency>
+            <groupId>org.openjfx</groupId>
+            <artifactId>javafx-controls</artifactId>
+            <version>11.0.2</version>
+        </dependency>
+        <dependency>
+            <groupId>org.openjfx</groupId>
+            <artifactId>javafx-fxml</artifactId>
+            <version>11.0.2</version>
+        </dependency>
+        <dependency>
+            <groupId>com.gluonhq</groupId>
+            <artifactId>charm-cloudlink-client</artifactId>
```

Comment:
@tiainen Can we release this artifact in the public-snapshots repository. Once done, we can get rid of the private snapshot repository and everyone can run the demo.

Reply:
@abhinayagarwal I've published 6.0.0-SNAPSHOT in the Gluon public snapshots repository:
https://nexus.gluonhq.com/nexus/content/repositories/public-snapshots/